forge_tests/TestOperatorStore.sol

```diff
@@ -0,0 +1,158 @@
+            duration: 0,
+            weight: 0,
+            discountRate: 0,
+            ballot: IJBFundingCycleBallot(address(0))
+        });
+
+        _metadata = JBFundingCycleMetadata({
+            global: JBGlobalFundingCycleMetadata({
+                allowSetTerminals: false,
+                allowSetController: false,
+                pauseTransfers: false
+            }),
+            reservedRate: 0,
+            redemptionRate: 0,
+            baseCurrency: uint32(uint160(JBTokens.ETH)),
+            pausePay: false,
+            allowMinting: false,
+            allowTerminalMigration: false,
+            allowControllerMigration: false,
+            holdFees: false,
+            useTotalOverflowForRedemptions: false,
+            useDataSourceForPay: false,
+            useDataSourceForRedeem: false,
+            dataSource: address(0),
+            metadata: 0
+        });
+
+        // Package up cycle config.
+        JBFundingCycleConfig[] memory _cycleConfig = new JBFundingCycleConfig[](1);
+        _cycleConfig[0].mustStartAtOrAfter = 0;
+        _cycleConfig[0].data = _data;
+        _cycleConfig[0].metadata = _metadata;
+        _cycleConfig[0].groupedSplits = new JBGroupedSplits[](0);
+        _cycleConfig[0].fundAccessConstraints = new JBFundAccessConstraints[](0);
+
+        // Package up terminal config.
+        JBTerminalConfig[] memory _terminalConfigurations = new JBTerminalConfig[](1);
+        JBAccountingContextConfig[] memory _accountingContexts = new JBAccountingContextConfig[](1);
+        _accountingContexts[0] =
+            JBAccountingContextConfig({token: JBTokens.ETH, standard: JBTokenStandards.NATIVE});
+        _terminalConfigurations[0] =
+            JBTerminalConfig({terminal: _terminal, accountingContextConfigs: _accountingContexts});
+
+        _projectZero = _controller.launchProjectFor({
+            owner: makeAddr("8008"),
+            projectMetadata: _projectMetadata,
+            fundingCycleConfigurations: _cycleConfig,
+            terminalConfigurations: _terminalConfigurations,
+            memo: ""
+        });
+
+        _projectOne = _controller.launchProjectFor({
+            owner: _projectOwner,
+            projectMetadata: _projectMetadata,
+            fundingCycleConfigurations: _cycleConfig,
+            terminalConfigurations: _terminalConfigurations,
+            memo: ""
+        });
+    }
+
+    function testFailMostBasicAccess() public {
+        // Package up cycle config.
+        JBFundingCycleConfig[] memory _cycleConfig = new JBFundingCycleConfig[](1);
+        _cycleConfig[0].mustStartAtOrAfter = 0;
+        _cycleConfig[0].data = _data;
+        _cycleConfig[0].metadata = _metadata;
+        _cycleConfig[0].groupedSplits = new JBGroupedSplits[](0);
+        _cycleConfig[0].fundAccessConstraints = new JBFundAccessConstraints[](0);
+        
+        vm.prank(makeAddr("8008"));
+        _controller.reconfigureFundingCyclesOf(_projectOne, _cycleConfig, "");
```

Comment:
no assertion?

Reply:
Felt implicit to me as if reconfigure fails, as will the test, but ofc could fail for various reasons (if misconfigured).

What is the best way for readability though? First thought is assertEq(configured == block.timestamp).